contracts/utils/LERC20.sol

```diff
@@ -114,6 +114,7 @@ contract LERC20 is Context, ILERC20 {
     }
 
     function proposeLosslessTurnOff() override external onlyRecoveryAdmin {
+        require(losslessTurnOffTimestamp != type(uint256).max, "LERC20: TurnOff already proposed");
```

Comment:
I think we would need to check if it's not zero?

Reply:
CVF-163 not fixed?

Reply:
When the proposal gets executed we set `losslessTurnOffTimestamp` to max meaning that there are no other proposals as per the auditors recommendation.  So in order to know that there is a proposal, the timestamp has to be different than max. If we check for `!= 0` if there is any other timestamp (or the max timestamp) it would take it as there is a proposal. 

Regarding CVF-163, this checks weren't in place before so now they cannot propose to turn lossless off if it's already off or the other way around. This would make the event emit only when there's a change.

Reply:
But we could set it to 0 and later on check it if it's not a zero? That would reduce gas costs.

Reply:
Yes, personally I think it'd be more readable even setting it to zero. This was only set to max per the audit guidelines.

Changed to zero on commit bb7fe10